eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState.hs

```diff
@@ -106,7 +107,6 @@ module Cardano.Ledger.Shelley.LedgerState (
   epochStateStakeDistrL,
   epochStateUMapL,
   epochStatePoolParamsL,
-  epochStateRegDrepL,
```

Comment:
Removal of this lens as well as `vsDRepsL` and `vsCommitteeStateL` need to be noted in the changelog